<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloaderWidget</class>
 <widget class="QDialog" name="DownloaderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download Plugins...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="downloadButton">
     <property name="text">
      <string>Download Selected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTableWidget" name="repoTable"/>
   </item>
   <item row="0" column="1">
    <widget class="QTextBrowser" name="readmeBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
